recipes-qt/qt5/qtwebkit_git.bb

```diff
@@ -12,10 +12,10 @@ DEPENDS += "qtbase qtdeclarative icu ruby-native sqlite3 glib-2.0 libxslt gperf-
 # Patches from https://github.com/meta-qt5/qtwebkit/commits/b5.13
 # 5.13.meta-qt5.1
 SRC_URI += "\
-    file://0002-Do-not-skip-build-for-cross-compile.patch \
-    file://0003-Fix-build-with-non-glibc-libc-on-musl.patch \
-    file://0004-Fix-build-bug-for-armv32-BE.patch \
-    file://0005-PlatformQt.cmake-Do-not-generate-hardcoded-include-p.patch \
+    file://0001-Do-not-skip-build-for-cross-compile.patch \
+    file://0002-Fix-build-with-non-glibc-libc-on-musl.patch \
+    file://0003-Fix-build-bug-for-armv32-BE.patch \
+    file://0004-PlatformQt.cmake-Do-not-generate-hardcoded-include-p.patch \
     file://0006-Fix-build-with-bison37.patch \
```

Comment:
The commit message doesn't match with what this patch is doing.

Reply:
There was error my commit message in HEAD. These are from 5.15.2, not 5.12.2. Sorry about it.

I will check also changes in commit. I am updating QT branch to github first time.  Content of pull-request should be sync with QT repository.

Reply:
It seems that "QTWebKit: Fix build error on QT-5.15 branch" is only renaming patches. Other changes from original commit are already in master.

Reply:
This commit is now squashed in latest update.

Reply:
It is now updated via https://github.com/meta-qt5/meta-qt5/pull/377/commits/3084c01dd0ee64040df41f18fda5e3bb819a35d6